app/main.py

```diff
@@ -1,34 +1,78 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+        self.shape = u"\u25A1"
+
+    def __eq__(self, other: tuple) -> bool:
+        return other == (self.row, self.column)
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
+            end: tuple,
+            is_drowned: bool = False
+    ) -> None:
+        self.decks = self.ship(start, end)
+        self.is_drowned = is_drowned
+
+    def __len__(self) -> int:
+        return len(self.decks)
+
+    @staticmethod
+    def ship(start: tuple, end: tuple) -> list:
```

Comment:
The purpose of this method is not clear from its name, choose another method name that should include a particular action that is done by it.